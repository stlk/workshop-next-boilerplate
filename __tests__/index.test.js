import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../pages/index.js';

describe('index', () => {
  it('Index shows gists', () => {
    const gists = [
      {
        id: 'gist-1',
        files: {
          file: {
            filename: 'filename'
          }
        }
      }
    ];
    const profile = {
      name: 'next guy',
      avatar_url: 'url'
    };
    const component = renderer.create(
      <Index gists={gists} profile={profile} />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
