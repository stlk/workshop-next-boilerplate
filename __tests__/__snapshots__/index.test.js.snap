// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Index shows gists 1`] = `
<div
  className="Layout-e2ivzo-1 iiMqQr"
>
  <div
    className="Layout__Content-e2ivzo-0 grYFab"
  >
    <p>
      This is the index page
    </p>
  </div>
</div>
`;
